### PR TITLE
fix munmap bug in the windows

### PR DESCRIPTION
**Pull request by runner-mei**
fix munmap bug in the windows

### CODE REVIEW

File: tsdb/engine/tsm1/mmap_windows.go [hunk 1]
```diff
@@ -99,7 +98,9 @@ func munmap(b []byte) (err error) {
 	delete(handleMap, addr)
 
 	e := syscall.CloseHandle(syscall.Handle(handle))
-	return os.NewSyscallError("CloseHandle", e)
+	if nil != e {
```
Comment (corylanou) [inline tsdb/engine/tsm1/mmap_windows.go]: nit: `if e != nil {`

File: tsdb/engine/tsm1/mmap_windows.go [hunk 2]
```diff
@@ -109,7 +110,8 @@ func munmap(b []byte) (err error) {
 	delete(fileMap, addr)
 
 	e = file.Close()
-	return errors.New("Close File" + e.Error())
-
+	if nil != e {
```
Comment (corylanou) [inline tsdb/engine/tsm1/mmap_windows.go]: nit `if e != nil {`

File: tsdb/engine/tsm1/mmap_windows.go [hunk 3]
```diff
@@ -109,7 +110,8 @@ func munmap(b []byte) (err error) {
...
+		return errors.New("Close File" + e.Error())
```
Comment (corylanou) [inline tsdb/engine/tsm1/mmap_windows.go]: errors should all be lower case `close file`

**Comment by corylanou** [comment]
Thank you for the PR!  Looks good other than some minor nits I would like to see changed before merging.

**Comment by corylanou** [comment]
Awesome! +1 Thanks!

**Comment by mvadu** [comment]
+1 @runner-mei can't believe I did those mistakes :-)

**Pull request merged**